### PR TITLE
Create CardListComponent #12

### PR DESCRIPTION
**Pull request by SunnyJinny**
- Create Component Card-List (for Desktop, not yet Mobile) 
- Add common design specs in Styles.scss
- Add bootstrap-Icon CDN in Index.html
- Add Dummy Data in Models

https://user-images.githubusercontent.com/53663147/205103133-cddfab26-38bc-4adb-890a-36129f54e66b.mov

### CODE REVIEW

**Comment by SunnyJinny** [comment]
Add new Card (todo: after Search hidden)

File: frontend/src/app/card-list/card-list.component.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<div class="container-fluid">
+  <ul class="cardList-wrapper"> <!-- grid -->
+    <!-- todo: after search hidden -->
+    <a href="" class="newBagel" *ngIf="!searched">
+      <div>+ new card</div>      
+    </a>
+    <a href="" class="cardItem" *ngFor="let bagel of bagels"> <!-- flex -->
```
Comment (Tiggershine) [inline frontend/src/app/card-list/card-list.component.html:7]: TODO: 
TrackBy 추가

**Comment by SunnyJinny** [comment]
remove bookmark

**Review by Tiggershine — APPROVED** [review]
Approve!

**Pull request merged**